packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/LinkList.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import {
+  Link as JssLink,
+  Text,
+  LinkField,
+} from '@sitecore-jss/sitecore-jss-nextjs';
+import { TextField } from '@sitecore-jss/sitecore-jss-react';
+
+type FieldLink = {
+  field: {
+    link: LinkField;
+  }
+};
+
+interface Fields {
+  data: {
+    datasource: {
+      children: {
+        results: Array<FieldLink>
+      };
+      field: {
+        title: TextField;
+      };
+    };
+  };
+};
+
+type LinkListProps = {
+  params: { [key: string]: string };
+  fields: Fields;
+};
+
+type LinkListItem = {
+  index: number;
+  key: string | number;
+  total: number;
+  field: LinkField;
+};
+
+const LinkListItem = (props: LinkListItem) => {
+  let className = `item${props.index}`;
+
+  className += (props.index + 1) % 2 == 0 ? ' odd' : ' even';
+
+  if (props.index == 0) {
+    className += ' first';
+  }
+
+  if (props.index + 1 == props.total) {
+    className += ' last';
+  }
+
+  return (
+    <li className={className}>
+      <div className="field-link">
+        <JssLink field={props.field} />
+      </div>
+    </li>
+  );
+};
+
+export const Default = (props: LinkListProps): JSX.Element => {
+  const datasource = props.fields?.data?.datasource;
+  const list = datasource?.children?.results
+    .filter(item => item.field.link)
+    .map((item: FieldLink, key: number) => (
+      <LinkListItem
+        index={key}
+        key={key}
```

Comment:
In `react` world it's not recommended to use `index` as a `key` for the item in the array. For example you can set `key` as an `item.field.link` value +`key`